main/src/cgeo/geocaching/connector/trackable/AbstractTrackableConnector.java

```diff
@@ -18,6 +18,9 @@
 public abstract class AbstractTrackableConnector implements TrackableConnector {
 
     @Override
+    public abstract int getPreferenceActivity();
```

Comment:
Overriding an interface method as an abstract method in a class doesn't make sense (and is reported as code inspection issue in IDEA at least). I suggest to make this a concrete method and to remove the all the "return 0;" implementations from the other connectors.